experiment/posttest.json

```diff
@@ -52,7 +52,7 @@
             "difficulty": "intermediate"
         },
         {
-            "question": "5. If two of the three outputs is known third one can be infered:",
+            "question": "5. If two out of the three outputs are known third one can be infered:",

```

Comment:
Inferred in place of infered . There should be a comma after known:
If two out of the three outputs are known, the third can be inferred